Documentation/gettingstarted/clustermesh.rst

```diff
@@ -29,8 +29,12 @@ Prerequisites
 * All nodes must have a unique IP address assigned them. Node IPs of clusters
   being connected together may not conflict with each other.
 
-* Cilium must be configured to use etcd as the kvstore. Consul is not supported
-  by cluster mesh at this point.
+* Cilium must be configured to use etcd as the kvstore. Also the default
+  settings of ``global.identityAllocationMode`` should be changed. With direct
+  etcd connections, for identities to be propagated across the clusters and
+  for cross-cluster policy to work, it should be set to ``kvstore`` as shown
+  ``--set global.identityAllocationMode=kvstore``. Consul is not supported by
+  cluster mesh at this point.
```

Comment:
```suggestion
* Cilium must be configured to use etcd as the kvstore, along with the identity allocation mode
(``global.identityAllocationMode``). With the identity allocation mode set to ``kvstore`` as shown, this 
allows direct etcd connections, identity porpgagation across the clusters, and enables cross-cluster policy
functionality. Consul is not supported by cluster mesh at this point.
```

I think this reads better, but let me know what you think. Feel free to tinker with it.

Reply:
If we're going to state the Helm option for the identity allocation mode, I think it also makes sens to state the helm option for setting the kvstore to etcd.

Reply:
Also I don't see the reference for `as shown`. I think those two words can be removed.

And `propagation` spelling :-)

`Consul is not supported by cluster mesh at this point` -> `Consul is not currently supported with Cluster Mesh`. Incidentally we should ensure that all references to the clustermesh feature consistently use the same capitalization / spacing)

Reply:
Ah yes `porpgagation` is used when `global.derpMode.enabled=true` :joy:

Reply:
Will address your comments.

Reply:
> If we're going to state the Helm option for the identity allocation mode, I think it also makes sens to state the helm option for setting the kvstore to etcd.

I think the helm option for kvstore is set when 'global.etcd' is enabled. Do you want to capture it here.

Reply:
Done.